crates/uv/src/commands/pip_install.rs

```diff
@@ -173,28 +175,46 @@ pub(crate) async fn pip_install(
     // If the requirements are already satisfied, we're done. Ideally, the resolver would be fast
     // enough to let us remove this check. But right now, for large environments, it's an order of
     // magnitude faster to validate the environment than to resolve the requirements.
-    if reinstall.is_none()
-        && upgrade.is_none()
-        && source_trees.is_empty()
-        && overrides.is_empty()
-        && site_packages.satisfies(&requirements, &editables, &constraints)?
-    {
-        let num_requirements = requirements.len() + editables.len();
-        let s = if num_requirements == 1 { "" } else { "s" };
-        writeln!(
-            printer.stderr(),
-            "{}",
-            format!(
-                "Audited {} in {}",
-                format!("{num_requirements} package{s}").bold(),
-                elapsed(start.elapsed())
-            )
-            .dimmed()
-        )?;
-        if dry_run {
-            writeln!(printer.stderr(), "Would make no changes")?;
+    if reinstall.is_none() && upgrade.is_none() && source_trees.is_empty() && overrides.is_empty() {
+        match site_packages.satisfies(&requirements, &editables, &constraints)? {
+            SatisfiesResult::Fresh {
+                recursive_requirements,
+            } => {
+                if enabled!(Level::DEBUG) {
+                    for requirement in recursive_requirements
+                        .iter()
+                        .map(ToString::to_string)
+                        .sorted()
+                    {
+                        debug!("Requirement satisfied: {requirement}");
```

Comment:
Can we make these ones trace?

Reply:
I guess I'm actually okay with debug.

Reply:
These should stay debug messages, they are similar to the message about the packages that we installed. They are the nearly final messages (this is the early exist path), they are only followed by:

```
DEBUG All editables satisfied: [...]
Audited 1 package in 12ms
```